validator/client/attest_protect.go

```diff
@@ -21,13 +22,18 @@ func (v *validator) preAttSignValidations(ctx context.Context, indexedAtt *ethpb
 	v.attesterHistoryByPubKeyLock.RLock()
 	attesterHistory, ok := v.attesterHistoryByPubKey[pubKey]
```

Comment:
This just holds a reference to the history, if another routine is also writing it at the same time, you will end up with conflicting histories.   The reference isn't threadsafe.

Reply:
implemented the change
will test as you suggested as soon as i get victor to stop running his keys